ui/raidboss/data/03-hw/dungeon/gubal_library_hard.js

```diff
@@ -107,69 +118,85 @@
       },
       infoText: {
         en: 'Stand in blue',
+        fr: 'Restez dans le bleu',
       },
     },
     {
       // This inflicts a vulnerability stack on the tank if not interrupted
       id: 'Gubal Hard Condensed Libra',
       regex: / 14:198D:Mechanoscribe starts using Condensed Libra on \y{Name}/,
+      regexFr: / 14:198D:Mécano-scribe starts using Acuité [cC]oncentrée on \y{Name}/,
```

Comment:
You don't need the brackets anymore, by the way.  All regexes are now case insensitive.  :)

Reply:
I'm going to merge this anyway, as it's good to have translations and there are a lot of these things to clean up so a few more isn't awful.

Reply:
Ok, that's a fantastic news :)